src/sparse_ir/kernel.py

```diff
@@ -168,6 +176,14 @@ def get_symmetrized(self, sign):
     @property
     def conv_radius(self): return 40 * self.lambda_
 
+    def weight_func(self, statistics: str):
+        """ Return the weight function for given statistics """
+        assert statistics in "FB"
+        if statistics=="F":
+            return lambda y: np.ones_like(y)
+        else:
+            return lambda y: 1/np.tanh(0.5*y)
```

Comment:
Is that correct?  I thought this was simply a factor of `y`?

Reply:
Ah! I am stupid, I got confused as to which kernel this belongs to! You are of course right about comment #11 ... shall we add this here?